atomix/cluster/src/main/java/io/atomix/raft/protocol/ConfigureRequest.java

```diff
@@ -39,18 +40,62 @@ public class ConfigureRequest extends AbstractRaftRequest {
   private final long index;
   private final long timestamp;
   private final Collection<RaftMember> members;
+  private final Collection<RaftMember> oldMembers;
```

Comment:
Just to be on same page : It is ok to add new fields, because we use `CompatibleFieldSerializer` with `chunkedEncoding = true`. Right?

Reply:
Yes exactly :+1: And the existing field needs to keep the old name so I couldn't rename `members` to `newMembers`

Reply:
All of these is covered by our upgrade tests, they started failing as soon as I tried renaming `members` to `newMembers`.